packages/data/package.json

```diff
@@ -23,10 +23,12 @@
   },
   "dependencies": {
     "ember-ajax": "^5.0.0",
+    "ember-auto-import": "^1.2.19",
     "ember-cli-babel": "^7.11.1",
     "ember-cli-htmlbars": "^4.0.0",
     "ember-get-config": "0.2.4",
-    "ember-lodash": "^4.19.5"
+    "ember-lodash": "^4.19.5",
+    "papaparse": "^5.1.1"
```

Comment:
Should we add this to the host `devDependencies`?

Reply:
It is pulled in via `ember-auto-import` so they both must be in `dependencies` otherwise [it won't work](https://github.com/ef4/ember-auto-import#usage-from-addons), unfortunately